examples/apps/colorapp/README.md

```diff
@@ -319,7 +319,7 @@ $ export AWS_DEFAULT_REGION=us-west-2
 $ export ENVIRONMENT_NAME=DEMO
 $ export SERVICES_DOMAIN=demo.local
 $ export KEY_PAIR_NAME=tony_devbox2
-$ export ENVOY_IMAGE=111345817488.dkr.ecr.$AWS_DEFAULT_REGION.amazonaws.com/aws-appmesh-envoy:v1.11.1.1-prod
+$ export ENVOY_IMAGE=12345689012.dkr.ecr.us-west-2.amazonzaws.com/appmesh-envoy:version
```

Comment:
Actually, I wonder if we should just replace these with the text like in the other places.